modules/create_book.rb

```diff
@@ -0,0 +1,13 @@
+module CreateBook
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    books << book
+    puts 'Book created succesfully.'
+    puts "\n"
+  end
```

Comment:
- [ ] All input and input validation requests should've been placed in the main.rb file and/or classes/methods which are responsible for requesting or validating input. please have another look at [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) which explains the best practices and the solid principle.